index.html

```diff
@@ -15,15 +15,148 @@
+          from Montreal, living in LA.
+        When I'm not working, I'm probably drawing,
+        binge watching the latest hot series,
+        or taking photos of my cats</p></div>
+        <div class="paragraph-square">
+          <h3 class="body-titles">Expertise</h3>
+          <p>My work tends to be end-to-end product design and photography.
+         I've supported teams as a senior IC for years,
+         leading strategy and vision for complex systems, 
+         as well as strengthening culture and team growth.
+          </p></div>
+        <div class="paragraph-square">
+          <h3 class="body-titles">Work with me</h3>
+          <p>Shoot me a message below if you'd like to talk photography or design.
+             Here's my résumé.</p>
+        </div> 
+    </div>
+    
+    <div class="wavy-line">
+      <img src="wavy_line.png" alt="wavy line"><!--GO BACK AND CENTER THE LINE :(-->
+    </div>
+    </section>
+    <section>
+    <div class="Portfolio-container">
+      <h3 class="T-Portfolio">Portfolio</h3>
+      <p class="p-intro">Welcome to Lisa Jonas's Portfolio site. I am a newly graduated Photographer and Designer. 
+        I've found passion in capturing all of the smallest details that make our world a better place.</p>
+    </div>
+    <div class="portfolio-photos">
+      <img src="1.jpeg" alt="Photo 1">
+      <img src="2.jpeg" alt="Photo 2">
+      <img src="3.jpeg" alt="Photo 3">
+      <img src="4.jpeg" alt="Photo 4">
+    </div>
+    <div class="expand">
+      <img src="expand.png" alt="expand button">
+    </div>
+    <div class="expand-more">
+      <p>Expand to view more</p>
+    </div>
+    <div class="wavy-line">
+      <img src="wavy_line.png" alt="wavy line"><!--GO BACK AND CENTER THE LINE :(-->
+    </div>
+  </section>
+  <section>
+    <div class="get-your-own">
+      <p class="title-gyo">Get your own MeLisa wall art</p>
+    </div>
+    <div class="store">
+      <img src="store.jpg" alt="store">
+    </div>
+    <div class="expand">
+      <img src="expand.png" alt="expand button">
+    </div>
+    <div class="expand-more">
+      <p>Expand to view more</p>
+    </div>
+    <div class="wavy-line">
+      <img src="wavy_line.png" alt="wavy line"><!--GO BACK AND CENTER THE LINE :(-->
+    </div>
+  </section>
+  <section>
+    <div class="t-blog">
+      <p>Blog</p>
+    </div>
+    <div class="p-blog">
+      <p class="p-blog">I am so happy to be able to share with you my journey in life, therefor, I will be
+        updating my weekly blog with all the things that make me happy!</p>
+    </div>
+    <div class="blog-photos">
+      <img src="blog1.jpg" alt="blog 1">
+      <img src="blog2.jpg" alt="blog 2">
+    </div>
+    <div class="wavy-line">
+      <img src="wavy_line.png" alt="wavy line"><!--GO BACK AND CENTER THE LINE :(-->
+    </div>
+  </section>
+</body>
+  <footer>
+    <section>
+    <div class="contact">
+      <p>Contact</pa>
+    </div>
+    <div class="contact-form">
```

Comment:
This "Contact" paragraph should be marked up as a `hX` heading (whatever level is appropriate - h2, h3, h4, etc.)